src/formatDate.js

```diff
@@ -50,7 +50,54 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
```

Comment:
Use destructuring to extract values from arrays. Example: const [day, month, year] = dateArr;

Reply:
the fact is that destructuring is not appropriate here, since we cannot say exactly in what order the hour, month, and day go